### DISCUSSION TITLE
Question about encryption on postal

### DISCUSSION THREAD

**Discussion by Sporesirius** [postalserver/postal#1613] [Help with using Postal]
Hello,

so, I installed postal and configured postal to use TLS.

Like:
>  port: 587
>   tls_enabled: true
>   tls_certificate_path: /opt/postal/app/certs/acme.zerossl.com-v2-dv90/mail.###/mail.###.crt
>   tls_private_key_path: /opt/postal/app/certs/acme.zerossl.com-v2-dv90/mail.###/mail.###.key
>   tls_ciphers:
>   ssl_version: TLSv1.2
>   proxy_protocol: false
>   log_connect: true
>   strip_received_headers: false
>   max_message_size: 14 # size in Megabytes

Is it possible to force postal to only use TLS and no STARTTLS, or Plain?

Kind regards

**Comment by willpower232** [comment]
It is not currently possible to disable plain authentication without changing the code to remove the options.

**Reply by Sporesirius** [reply]
Is that on the to-do list?

**Reply by willpower232** [reply]
not to my knowledge

**Reply by Sporesirius** [reply]
Isn't something like that a high priority? Or does that not matter for E-Mails?

**Reply by willpower232** [reply]
It certainly doesn't sound like something that would be on Postals priority list at the minute.

I imagine that, historically, devices which can send SMTP emails may not always have enough computing capacity available to handle the TLS so it has to be an "opt in" feature.
